vms/platformvm/state/state.go

```diff
@@ -181,13 +214,9 @@ type stateBlk struct {
  * | |   '-. list
  * | |     '-- txID -> nil
  * | |-. weight diffs
- * | | '-. height+subnet
```

Comment:
Do we want to keep this old layout around given that we aren't migrating it away?